File: src/Python/modules/application/weather_text_mapper.py
from modules.domain.user_cases.weather.weater_model import WeatherModel


class WeatherTextMapper:
    @staticmethod
    def get_weather_text(weather: WeatherModel) -> str:
        if not weather.is_raining:
            return "WTF ! What is this fireball on the sky ?!"

        return "It's raining, of course we are in britany"

File: src/C#/Pillont.Shodo.Application.Console/WeatherTextMapper.cs
using Pillont.Shodo.Domain.UseCases.Weather.Models;

namespace Pillont.Shodo.Application.Console;
public static class WeatherTextMapper
{
    public static string GetWeatherText(WeatherModel weather)
    {
        return weather.IsRaining
            ? "It's raining, of course we are in britany"
            : "WTF ! What is this fireball on the sky ?!";
    }
}
